Documentation/installation/cni-chaining-azure-cni.rst

```diff
@@ -6,17 +6,15 @@
 
 .. _chaining_azure:
 
-*********
-Azure CNI
-*********
+******************
+Azure CNI (Legacy)
+******************
 
 .. note::
-
-   For most users, the best way to run Cilium on AKS is either
-   AKS BYO CNI as described in :ref:`k8s_install_quick`
-   or `Azure CNI Powered by Cilium <https://aka.ms/aks/cilium-dataplane>`_.
+   The best way to run cilium on AKS is through :ref:`aks_install` or `Azure CNI Powered by Cilium <https://aka.ms/aks/cilium-dataplane>`_.
    This guide provides alternative instructions to run Cilium with Azure CNI
-   in a chaining configuration.
+   in a chaining configuration which is the legacy way of running Azure CNI with
```

Comment:
nit: seems like this could be two sentences ... i.e. in chaining configuration.  This is the legacy way...